experimental/streaming/exporter/observer/observer.go

```diff
@@ -152,6 +154,9 @@ func Foreach(f func(Observer)) {
 }
 
 func NewScope(parent ScopeID, attributes ...core.KeyValue) ScopeID {
+	if len(attributes) == 0 {
```

Comment:
Are we going to get an empty `ScopeID` if we pass empty `parent` and no attributes?